.github/workflows/publish_on_request.yml

```
@@ -0,0 +1,105 @@
+name: Publish on request
+
+on:
+  issue_comment:
+    types: [created, edited]
+
```

We just realize that the build job needs the GITHUB_TOKEN to fetch languages !!! And it's missing in the files.
Could you add this few lines 🙂
```suggestion
env:
  GIT_TOKEN: ${{ secrets.GIT_TOKEN }}
  
```